plugins/inputs/pgbouncer/pgbouncer.go

```diff
@@ -130,7 +134,55 @@ func (p *PgBouncer) Gather(acc telegraf.Accumulator) error {
 		acc.AddFields("pgbouncer_pools", fields, tags)
 	}
 
-	return poolRows.Err()
+	err = poolRows.Err()
```

Comment:
I hesitate to accept this PR because it forces all plugin users to get new database metrics even when they may not want or need them.

What do you think about adding a plugin setting for this functionality? There are lots of show commands besides `show stats`, `show pools`, `show databases`. I would like a setting that lists which sections the user wants so they don't get metrics (and overhead to gather the metrics) when they aren't going to use them.

The setting should default to the previous behavior of `show stats` and `show pools` but let people add or remove which sections they want.

```toml
## Specify which "show" commands to gather metrics for. Valid options: "stats", "pools", "databases"
#show_commands = ["stats", "pools"]
```

Then in the future we can add "servers", "clients", "lists", "users", etc. as needed